config/routes.rb

```diff
@@ -13,4 +15,5 @@
 
   resources :users
   resources :account_activations, only: :edit
+  resources :password_resets, only: [:new, :create, :edit, :update]
```

Comment:
should use `except %i(index show delete) ` instead of `only: [:new, :create, :edit, :update]`